src/index.js

```diff
@@ -85,35 +86,28 @@ function format(options) {
     })
   );
 
-  const isCss = /\.(css)$/.test(filePath);
-  const isLess = /\.(less)$/.test(filePath);
-  const isScss = /\.(scss)$/.test(filePath);
-  const isTypeScript = /\.(ts|tsx)$/.test(filePath);
-  const isGraphQL = /\.(graphql|gql)$/.test(filePath);
-  const isJson = /\.json$/.test(filePath);
+  const eslintExtensions = eslintConfig.extensions || [
+    ".js",
+    ".jsx",
+    ".ts",
+    ".tsx"
+  ];
+  const eslintExtensionsRegex = new RegExp(eslintExtensions.join("|"), "gi");
 
-  if (isCss) {
-    formattingOptions.prettier.parser = "css";
-  } else if (isLess) {
-    formattingOptions.prettier.parser = "less";
-  } else if (isScss) {
-    formattingOptions.prettier.parser = "scss";
-  } else if (isTypeScript) {
-    formattingOptions.prettier.parser = "typescript";
+  // If we don't get filePath run eslint on text, otherwise only run eslint
+  // if it's a configured extension or fall back to a "supported" file type.
+  const onlyPrettier = filePath ? !eslintExtensionsRegex.test(filePath) : false;
+
+  if (/\.tsx?$/.test(filePath)) {
```

Comment:
We should use `getSupportInfo` here, aren't we?

Reply:
Nope, this is for ESLint and if that should use `typescript-eslint-parser`. I guess I could move it to after the `if(onlyPrettier)` block, to try and make that more clear?

Reply:
Right, thanks. Makes sense to move it I think.